spec/carrierwave-aws_spec.rb

```diff
@@ -45,5 +58,34 @@
       expect(uploader.aws_acl).to eq('private')
       expect(instance.aws_acl).to eq('public-read')
     end
+
+    it 'can be looked up from superclass' do
+      uploader.aws_acl = 'private'
+      instance = derived_uploader.new
+
+      expect(derived_uploader.aws_acl).to eq 'private'
+      expect(instance.aws_acl).to eq 'private'
+    end
+
+    it 'can be overridden on a class level' do
+      uploader.aws_acl = 'public-read'
+      derived_uploader.aws_acl = 'private'
+
+      base = uploader.new
+      expect(base.aws_acl).to eq 'public-read'
+
+      instance = derived_uploader.new
+      expect(instance.aws_acl).to eq 'private'
+    end
+
+    it 'can lookup even if none configured' do
```

Comment:
I think you've proven that there was a inheritance chain issue now. I'd rather not keep expectations against instance variables though. This particular test can simply be removed.